ppi.c

```diff
@@ -81,6 +81,17 @@ setup_ppi_variables(void)
         free(data);
     }
 
+    status = internal_set_variable(TCG2_PHYSICAL_PRESENCEFLAGSLOCK_NAME,
```

Comment:
How about guests that get upgraded from a previous CH release? How will they gain the new PPI variables in UEFI store?

Reply:
I assume this code runs everytime varstored is initialized and creates it if missing?

Reply:
Every boot it is initilised to 0.   In any weird situation that its missing, it assumes its locked, which wolud be the case for any VM thats got into the OS.